.eslintrc.js

```diff
@@ -142,6 +148,7 @@ module.exports = {
     "unicorn/prefer-string-starts-ends-with": "error",
     "unicorn/prefer-switch": "error",
     "unicorn/prefer-type-error": "error",
+    "@typescript-eslint/prefer-ts-expect-error": "error",
```

Comment:
Current rules are more or less ordered alphabetically. How about placing `@typescript/*` before other rules to simplify visual search?